### DISCUSSION TITLE
Sync Settings - conflicts with VSCode?

### DISCUSSION THREAD

**Discussion by rkrug** [posit-dev/positron#3892] [Q&A]
Hi

I would like to setup Sync Settings - what is the recommended way to avoid interference with VSCods Settings Sync?

**Comment by juliasilge** [comment]
The Sync Settings provided by VS Code uses Microsoft servers/telemetry/etc. Our builds of Positron are more like VS Codium than the Microsoft-provided builds of VS Code in that we have turned off all access to such servers and telemetry; see https://github.com/posit-dev/positron/pull/2882.

I have not tried it out myself, but you might [look at this open source alternative](https://open-vsx.org/extension/Shan/code-settings-sync) or choose another way that VS Codium users like to handle syncing their settings across devices.

**Comment by rkrug** [comment]
Thanks - yes. I understand why you turned of these access. 

It would be great if you could, at some time, supply a suggested / approved solution for syncing. But not that urgent.
